eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -107,6 +107,10 @@
     <Class name="~com\.microsoft\.azure\.(.+)Impl"/>
     <Bug pattern="NP_LOAD_OF_KNOWN_NULL_VALUE"/>
   </Match>
+  <Match>
+    <Class name="~com\.azure\.(.+)Impl"/>
```

Comment:
Use the `<Or>` pattern you can see elsewhere to specify multiple classes for a particular bug pattern